src/topology/algebra/monoid.lean

```diff
@@ -112,6 +112,10 @@ instance pi.has_continuous_mul {C : ι → Type*} [∀ i, topological_space (C i
 { continuous_mul := continuous_pi (λ i, continuous.mul
     ((continuous_apply i).comp continuous_fst) ((continuous_apply i).comp continuous_snd)) }
 
+@[to_additive]
+instance pi.has_continuous_mul' : has_continuous_mul (ι → M) :=
+pi.has_continuous_mul
```

Comment:
This could do with a comment explaining why it is needed too.

Maybe we need a library note for this kind of non-dependent variant?

Reply:
I added 2 comments. Feel free to write a library note.